include/circt/Dialect/FIRRTL/FIRRTLDeclarations.td

```diff
@@ -61,39 +61,95 @@ def InstanceOp : FIRRTLOp<"instance"> {
   }];
 }
 
-def CMemOp : FIRRTLOp<"cmem", [/*MemAlloc*/]> {
-  let summary = "Define a new cmem";
+def CombMemOp : FIRRTLOp<"combmem"> {
+  let summary = "Define a new combinational memory";
+  let description = [{
+    Define a new behavioral combinational memory. Combinational memories have a
+    write latency of 1 and a read latency of 0. This memory should be used by
```

Comment:
The sentence is hanging. 😄

Reply:
Removed! I keep wanting to put an example of a CHIRRTL memory in action, but I'm not sure where to put it since `cmem` and `smem` are different operations.